dist/build.gradle

```diff
@@ -153,7 +153,7 @@ task docker(group: 'Build',
             dependsOn: tasks.distDir,
             type: DockerBuildImage) {
     inputDir = file('.')
-    def name = 'line/centraldogma:'
+    def name = 'ghcr.io/line/centraldogma:'
```

Comment:
Question) I understood this task to just build the image locally, so I find it a little odd that the registry name is included in the image name. Was this change necessary?

Reply:
Without the registry prefix, the following command fails to find the target image.
```
docker push ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:latest 
```
So, it is necessary.